tools/api/generate_go_protobuf.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+from subprocess import check_output
```

Comment:
Nit: prefer these mixed with the `import` below in alpha order.

Reply:
Done